pkg/interface/src/views/apps/publish/components/NotebookRoutes.tsx

```diff
@@ -70,6 +71,9 @@ export function NotebookRoutes(
           const { noteId } = routeProps.match.params;
           const note = notebook?.notes[noteId];
           const noteUrl = relativePath(`/note/${noteId}`);
+          if(!note) {
+            return <Center height="100%"><LoadingSpinner /></Center>;
+          }
```

Comment:
The only thing in this commit is spinner behaviour for no note case? But the note never loads in, last I looked